stubs/pywin32/isapi/install.pyi

```diff
@@ -56,43 +59,43 @@ class ISAPIParameters:
 
 verbose: int
 
-def log(level, what) -> None: ...
+def log(level: int, what: str) -> None: ...
 
 class InstallationError(Exception): ...
 class ItemNotFound(InstallationError): ...
 class ConfigurationError(InstallationError): ...
 
-def FindPath(options, server, name): ...
-def LocateWebServerPath(description): ...
-def GetWebServer(description: Incomplete | None = None): ...
+def FindPath(options, server: str | bytes, name: str) -> str: ...
+def LocateWebServerPath(description: str): ...
+def GetWebServer(description: str | None = None): ...
 def LoadWebServer(path): ...
-def FindWebServer(options, server_desc): ...
-def split_path(path): ...
+def FindWebServer(options, server_desc: str | bytes | None) -> str: ...
+def split_path(path: str) -> list[str]: ...
 def CreateDirectory(params, options): ...
-def AssignScriptMaps(script_maps, target, update: str = "replace") -> None: ...
+def AssignScriptMaps(script_maps: Iterable[ScriptMapParams], target, update: str = "replace") -> None: ...
 def get_unique_items(sequence, reference): ...
 def CreateISAPIFilter(filterParams, options): ...
 def DeleteISAPIFilter(filterParams, options) -> None: ...
 def AddExtensionFiles(params, options) -> None: ...
 def DeleteExtensionFileRecords(params, options) -> None: ...
-def CheckLoaderModule(dll_name) -> None: ...
+def CheckLoaderModule(dll_name: StrOrBytesPath) -> None: ...
 def Install(params, options) -> None: ...
 def RemoveDirectory(params, options) -> None: ...
 def RemoveScriptMaps(vd_params, options) -> None: ...
 def Uninstall(params, options) -> None: ...
-def GetLoaderModuleName(mod_name, check_module: Incomplete | None = None): ...
-def InstallModule(conf_module_name, params, options, log=...) -> None: ...
-def UninstallModule(conf_module_name, params, options, log=...) -> None: ...
+def GetLoaderModuleName(mod_name: StrPath, check_module: bool | None = None) -> str: ...
+def InstallModule(conf_module_name: StrPath, params, options, log: Callable[[int, str], None] = ...) -> None: ...
+def UninstallModule(conf_module_name: StrPath, params, options, log: Callable[[int, str], None] = ...) -> None: ...
 
-standard_arguments: Incomplete
+standard_arguments: dict[Literal["install", "remove"], Incomplete]
```

Comment:
your use of Literal to restrain the possible keys lead me to believe you're trying to indicate this dict shouldn't change. And I agree, let's go further and mark it as `Final` (this probably should be a frozendict at runtime)

I also indicated the value should be a Callable

```suggestion
standard_arguments: Final[dict[Literal["install", "remove"], Callable[..., Incomplete]]]
```